src/test/llm/visual_language_model/complete_flow_test.cpp

```diff
@@ -81,7 +81,7 @@ class VLMServableExecutionTest : public ::testing::Test {
 
 std::unique_ptr<std::thread> VLMServableExecutionTest::t;
 
-std::string createRequestBody(const std::string& modelName, const std::vector<std::pair<std::string, std::string>>& fields, bool includeText = true, bool includeImage = true) {
+static std::string createRequestBody(const std::string& modelName, const std::vector<std::pair<std::string, std::string>>& fields, bool includeText = true, int numberOfImages = 1, const std::string message = "What is in this image?") {
```

Comment:
rename `message -> prompt`?
in our context message is a bigger structure

Reply:
it is not a prompt either, should I name it contentOfTheFirstMessage?